pipeline/cvp.groovy

```diff
@@ -7,9 +7,10 @@ def nodeName = "centos-7"
 def tierLevel = "tier-0"
 def testStages = [:]
 def testResults = [:]
-def cliArgs = "--build cvp"
+def overrides = {}
 def sharedLib
 def jobStatus
+def tags = "openstack,tier-0"
```

Comment:
We don't need to run the entire suite for CVP... It is sufficient to execute only the build sanity test suite.